src/IIIFPresentation/API.Tests/Integration/ModifyCollectionTests.cs

```diff
@@ -464,4 +464,114 @@ public async Task UpdateCollection_FailsToUpdateCollection_WhenCalledWithoutNeed
         // Assert
         response.StatusCode.Should().Be(HttpStatusCode.Forbidden);
     }
+    
+    [Fact]
+    public async Task DeleteCollection_DeletesCollection_WhenAllValuesProvided()
```

Comment:
What does `WhenAllValuesProvider` mean here?

Reply:
`WhenAllValuesProvided` is meaning that everything has been provided that's needed to delete something successfully, like id, headers, etc